services/web/server/src/simcore_service_webserver/projects/_groups_handlers.py

```diff
@@ -0,0 +1,157 @@
+""" Handlers for project comments operations
+
+"""
+
+import functools
+import logging
+
+from aiohttp import web
+from models_library.projects import ProjectID
+from models_library.users import GroupID
+from pydantic import BaseModel, Extra
+from servicelib.aiohttp.requests_validation import (
+    parse_request_body_as,
+    parse_request_path_parameters_as,
+)
+from servicelib.aiohttp.typing_extension import Handler
+from servicelib.mimetype_constants import MIMETYPE_APPLICATION_JSON
+
+from .._meta import api_version_prefix as VTAG
+from ..login.decorators import login_required
+from ..security.decorators import permission_required
+from ..utils_aiohttp import envelope_json_response
+from . import _groups_api
+from ._common_models import ProjectPathParams, RequestContext
+from ._groups_api import ProjectGroupGet
+from .exceptions import ProjectGroupNotFoundError, ProjectNotFoundError
+
+_logger = logging.getLogger(__name__)
+
+
+def _handle_projects_groups_exceptions(handler: Handler):
+    @functools.wraps(handler)
+    async def wrapper(request: web.Request) -> web.StreamResponse:
+        try:
+            return await handler(request)
+
+        except ProjectGroupNotFoundError as exc:
+            raise web.HTTPNotFound(reason=f"{exc}") from exc
+
+        except ProjectNotFoundError as exc:
+            raise web.HTTPForbidden(reason=f"{exc}") from exc
+
+    return wrapper
+
+
+#
+# projects groups COLLECTION -------------------------
+#
+
+routes = web.RouteTableDef()
+
+
+class _ProjectsGroupsPathParams(BaseModel):
+    project_id: ProjectID
+    group_id: GroupID
+
+    class Config:
+        extra = Extra.forbid
+
+
+class _ProjectsGroupsBodyParams(BaseModel):
+    read: bool
+    write: bool
+    delete: bool
+
+    class Config:
+        extra = Extra.forbid
+
+
+@routes.post(
+    f"/{VTAG}/projects/{{project_id}}/groups/{{group_id}}", name="create_project_group"
```

Comment:
Suggestion: I saw in one of the other files that you managed to set the `"{VTAG}"` already when constructing the `router`, might be worth doing here too.